src/components/organisms/CatList/index.jsx

```diff
@@ -78,22 +78,26 @@ class CatList extends React.Component {
     },
   };
 
+  componentDidMount = () => {
+    this.props.getCats();
+  };
+
   componentWillReceiveProps(nextProps) {
-    // Check if cats have changed
-    if (JSON.stringify(this.props.cats) !== JSON.stringify(nextProps.cats)) {
-      nextProps.cats &&
-        this.setState({ sync: false }, () => this.fillTable(nextProps.cats));
+    // Check if users have changed
+    if (JSON.stringify(this.props.users) !== JSON.stringify(nextProps.users)) {
+      nextProps.users &&
+        this.setState({ sync: false }, () => this.fillTable(nextProps.users));
     } else {
       this.setState({ sync: false });
     }
   }
 
-  toggle = (cat) => {
+  toggle = (user) => {
     if (!this.state.modal) {
       this.setState({
         modal: true,
         modalCat: {
```

Comment:
Does it make sense to use all methods in connection with cats? 🐈
Or does cats use this website? I could be true.

Reply:
Cats are an entity in our customer acquisition workflow.

Reply:
Cute